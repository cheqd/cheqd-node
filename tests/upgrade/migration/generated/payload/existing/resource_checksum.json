{
    "collectionId": "z6LNquFfzPNhCQ8n",
    "id": "7f2589aa-1279-4bd7-b704-f868772b460a",
    "name": "Test Resource",
    "resourceType": "test-resource-type",
    "data": "{\"message\": \"hello world\"}"
}