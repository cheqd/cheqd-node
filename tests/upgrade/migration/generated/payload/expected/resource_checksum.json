{
    "collectionId": "z6LNquFfzPNhCQ8n",
    "id": "7f2589aa-1279-4bd7-b704-f868772b460a",
    "name": "Test Resource",
    "resourceType": "test-resource-type",
    "mediaType": "application/json",
    "created": "2022-11-01T10:54:01Z",
    "checksum": "4f479bba31f71859d9943651538c8df2119804eceb7f58a0de241789ff214af4",
    "previousVersionId": null,
    "nextVersionId": null
}